installer/resources/pacbot_app/files/DB.sql

```diff
@@ -1719,7 +1719,7 @@ INSERT IGNORE INTO cf_Policy (`policyId`,`policyName`,`policyDesc`,`resolution`,
 INSERT IGNORE INTO cf_Policy (`policyId`,`policyName`,`policyDesc`,`resolution`,`policyUrl`,`policyVersion`,`status`,`userId`,`createdDate`,`modifiedDate`,`resolutionUrl`) VALUES ('Enable_Integrity_Monitoring_For_Node_Pool','Enable Integrity Monitoring for Shielded GKE Nodes','Integrity monitoring should be enabled to monitor and check runtime boot integrity of shielded cluster nodes using Cloud Monitoring service','Once a Node pool is provisioned, it cannot be updated to enable Integrity Monitoring. You must create new Node pools within the cluster with Integrity Monitoring enabled Using Google Cloud Console \n1. Go to Kubernetes Engine by visiting https://console.cloud.google.com/kubernetes/list \n2. From the list of clusters, click on the cluster requiring the update and click ADD NODE POOL \n3. Ensure that the Integrity monitoring checkbox is checked under the Shielded options Heading. \n4. Click SAVE.',NULL,'version-1',NULL,NULL,'2022-10-26','2022-10-26','https://github.com/PaladinCloud/CE/wiki/GCP-Policy#Enable-Integrity-Monitoring-for-Shielded-GKE-Nodes');
 INSERT IGNORE INTO cf_Policy (`policyId`,`policyName`,`policyDesc`,`resolution`,`policyUrl`,`policyVersion`,`status`,`userId`,`createdDate`,`modifiedDate`,`resolutionUrl`) VALUES ('Enable_Secure_Boot_For_Node_Pool','Enable Secure Boot for Shielded GKE Nodes','Secure Boot should be enabled for shielded GKE nodes to ensure that the system only runs authentic software by verifying the digital signature of all boot components, and halting the boot  process if signature verification fails','Once a Node pool is provisioned, it cannot be updated to enable Secure Boot. You must create new Node pools within the cluster with Secure Boot enabled. Using Google Cloud Console \n1. Go to Kubernetes Engine by visiting https://console.cloud.google.com/kubernetes/list \n2. From the list of clusters, click on the cluster requiring the update and click ADD NODE POOL \n3. Ensure that the Secure boot checkbox is checked under the Shielded options Heading \n4. Click SAVE.',NULL,'version-1',NULL,NULL,'2022-10-26','2022-10-26','https://github.com/PaladinCloud/CE/wiki/GCP-Policy#Enable-Secure-Boot-for-Shielded-GKE-Nodes');
-INSERT IGNORE INTO cf_Policy (`policyId`,`policyName`,`policyDesc`,`resolution`,`policyUrl`,`policyVersion`,`status`,`userId`,`createdDate`,`modifiedDate`,`resolutionUrl`) VALUES ('Enable_rbac_for_azure_kubernetes','Enable RBAC for Azure Kubernetes Services','In Azure kubernetes services, role-based access control should be enabled','1. Login to Azure and navigate to Kubernetes Services.\n2. For each Kubernetes Services instance, click on Automation Script. \n3. Ensure that each variable enableRBAC is set to true.',NULL,'version-1',NULL,NULL,'2022-10-27','2022-10-27','https://github.com/PaladinCloud/CE/wiki/Azure-Policy#enable-role-based-access-control-rbac-within-azure-kubernetes-services');
```

Comment:
Please confirm once this policy change is correct.
For new policy it should be a new entry.

Reply:
this entry exists in db.sql dheeraj